theme_enark/__manifest__.py

```diff
@@ -13,7 +13,6 @@
         'views/image_library.xml',
 
         'views/snippets/s_banner.xml',
-        'views/snippets/s_cover.xml',
```

Comment:
Chances are that we'll have to add it back, for now let's not remove the file ;)